src/EnergyPlus/SQLiteProcedures.cc

```diff
@@ -2652,11 +2679,15 @@ SQLiteProcedures::SQLiteProcedures(std::shared_ptr<std::ostream> const &errorStr
             } else {
                 if (dbName != ":memory:") {
                     // Remove test db
-                    rc = remove(dbName.string().c_str());
-                    if (rc) {
-                        // File operation failed. SQLite connection is not in an error state.
-                        *m_errorStream << "SQLite3 message, can't remove old database." << std::endl;
-                        ok = false;
+                    // rc = remove(dbName_utf8.c_str());
+                    if (fs::is_regular_file(dbName)) {
+                        std::error_code ec;
+                        if (!fs::remove(dbName, ec)) {
+                            // File operation failed. SQLite connection is not in an error state.
+                            *m_errorStream << "SQLite3 message, can't remove old database. code=" << ec.value() << ", error: " << ec.message()
+                                           << std::endl;
+                            ok = false;
+                        }
```

Comment:
instead of calling the C stdio.h [remove](https://en.cppreference.com/w/c/io/remove), use the fs::remove one